gamemode/config/config.lua

```diff
@@ -107,6 +107,12 @@ GM.Config.lockpickfading                = true
 GM.Config.logging                       = true
 -- lottery - Enable/disable creating lotteries for mayors
 GM.Config.lottery                       = true
+-- lockdown - Enable/Disable initiating lockdowns for mayors.
+GM.Config.lockdown                      = true
+-- lockdownMaxDuration - Whether you want the lockdown to end automatically after a certain amount of time (the amount of time can be configured below).
+GM.Config.lockdownMaxDuration           = true
+-- preventLockdownSpam - Whether the Mayor needs to wait a certain amount of time before initiating another lockdown (the amount of time can be configured below).
+GM.Config.preventLockdownSpam           = true
```

Comment:
These two configs probably would just complicate things. How about just using the timer configs and taking 0 to be disabled?